libs/design/src/molecules/article/article/article-theme.scss

```diff
@@ -1,4 +1,4 @@
-@import './stops-article-cascade';
+@use 'stops-article-cascade' as *;
```

Comment:
Can we avoid the `*`?

Reply:
I didn't see a point in adding a namespace for it because the `stops-article-cascade` file is next to the theme and scss file and easily located. I can add a namespace if you want to completely avoid the `*`.